README.md

```diff
@@ -1,2 +1,121 @@
-# HelloMicroverse
-First Microverse Project
+<a name="readme-top"></a>
+
+<div align="center">
+
+<h3><b>First Project: Hello Microverse</b></h3>
+<br>
+<h4><b>Alejandro Orozco</b></h4>
+
+</div>
+<br>
+<br>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
```

Comment:
- [ ] Good job on the README 👏 . However, you missed some vital headers which are very important for your project.
- Some of which are `setup, usage, and installation`. These sections show your users or people of interest in your code how to run your project locally and use it.
- [ ] Under setup, you can run a guide like this:
```
cd folder
git clone < your repo http path>
code . (To open in Vscode)
```

## Expected Header:
![image](https://github.com/OROZCO1623/HelloMicroverse/assets/50990925/00f3f5a8-6feb-49fb-9f76-71d23fa67956)